core/chains/evm/client/errors.go

```diff
@@ -70,14 +70,16 @@ func (e *ClientErrors) ErrIs(err error, errorTypes ...int) bool {
 	if err == nil {
 		return false
 	}
+
+	cause := pkgerrors.Cause(err).Error()
```

Comment:
This actually seems brittle. Why don't we use the std lib here?
```suggestion
	cause := errors.Unwrap(err).Error()
```
pkgerrors will still return the same cause: https://github.com/pkg/errors/blob/master/errors.go#L163
and this won't break in the future if we wrap the errors a different way.